versionist.conf.js

```diff
@@ -0,0 +1,59 @@
+var execSync = require('child_process').execSync;
+
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
+
+module.exports = {
+  // This setup allows the editing and parsing of footer tags to get version and type information,
+  // as well as ensuring tags of the type 'v<major>.<minor>.<patch>' are used.
+  // It increments in a semver compatible fashion and allows the updating of NPM package info.
+  editChangelog: true,
+  parseFooterTags: true,
+  getGitReferenceFromVersion: 'v-prefix',
+  incrementVersion: 'semver',
+  updateVersion: (cwd, version, cb) => { cb(); },
```

Comment:
Rather than setting a 'no-op' `updateVersion`, perhaps it'd have been better to set `editVersion` to false? https://github.com/resin-io/versionist#editversion-boolean
(ping @hedss )